examples/circos/edge_color.py

```diff
@@ -0,0 +1,43 @@
+"""
+Shows different edge colors. Either categorial/ordinal or continuous
+"""
+import matplotlib.pyplot as plt
+import networkx as nx
+from nxviz.plots import CircosPlot
+
+nodelist = [('a'),
+                ('b'),
+                ('c'),
+                ('d'),
+                ('e'),
+                ('f')
+                ]
+edgelist1 = [('a', 'b', {'weight': 0.1}),
+            ('a', 'c', {'weight': 0.2}),
```

Comment:
`pycodestyle` is asking for the parentheses the be lined up. Please make this change.